cve_bin_tool/util.py

```diff
@@ -7,37 +7,41 @@
 import sys
 from collections import defaultdict
 from enum import Enum
-from typing import NamedTuple
+from typing import List, NamedTuple, Optional, Set, Tuple, Union
 
 
 class OrderedEnum(Enum):
-    def __ge__(self, other):
+    def __ge__(self, other) -> bool:
         if self.__class__ is other.__class__:
             return self.value >= other.value
         return NotImplemented
 
-    def __gt__(self, other):
+    def __gt__(self, other) -> bool:
         if self.__class__ is other.__class__:
             return self.value > other.value
         return NotImplemented
 
-    def __le__(self, other):
+    def __le__(self, other) -> bool:
         if self.__class__ is other.__class__:
             return self.value <= other.value
         return NotImplemented
 
-    def __lt__(self, other):
+    def __lt__(self, other) -> bool:
         if self.__class__ is other.__class__:
             return self.value < other.value
         return NotImplemented
 
 
+tup_4 = Tuple[int, str, str, str, str]
+tup_5 = Tuple[int, str, str, str, str, str]
+
+
 class Remarks(OrderedEnum):
-    NewFound = 1, "1", "NewFound", "n", "N"
-    Unexplored = 2, "2", "Unexplored", "u", "U", ""
-    Confirmed = 3, "3", "Confirmed", "c", "C"
-    Mitigated = 4, "4", "Mitigated", "m", "M"
-    Ignored = 5, "5", "Ignored", "i", "I"
+    NewFound: tup_4 = 1, "1", "NewFound", "n", "N"
+    Unexplored: tup_5 = 2, "2", "Unexplored", "u", "U", ""
+    Confirmed: tup_4 = 3, "3", "Confirmed", "c", "C"
+    Mitigated: tup_4 = 4, "4", "Mitigated", "m", "M"
+    Ignored: tup_4 = 5, "5", "Ignored", "i", "I"
 
     def __new__(cls, value, *aliases):
```

Comment:
```suggestion
    def __new__(cls, value: int, *aliases: str) -> Remarks:
```

Reply:
Thanks, I was not sure what the types of value and *aliases were, could only guess str from references of Remarks, so left that as is.